### ISSUE TITLE
Ambari::ShowMaster is failing on basic cluster:  Exception in thread "main" groovyx.net.http.HttpResponseException: Not Found

### ISSUE THREAD

**Issue by snowch** [snowch/biginsight-examples#75]
ShowMaster is failing on a basic cluster:

```
snowch$ cat build/test/Ambari-ShowMaster.txt 
:compileJava UP-TO-DATE
:compileGroovy
:processResources UP-TO-DATE
:classes
:ShowMaster

Exception in thread "main" groovyx.net.http.HttpResponseException: Not Found
    at groovyx.net.http.RESTClient.defaultFailureHandler(RESTClient.java:263)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
        ...
    at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:120)
    at ShowMaster.main(ShowMaster.groovy)
:ShowMaster FAILED
```

I can share credentials offline ...

**Comment by pregazzoni** [comment]
I suspect we are trying to get MASTER for a service that does not exists. Let me look at code, but i assume basic cluster does not have BIGSQL head for example.

**Comment by pregazzoni** [comment]
ok i confirmed we don't have BIGSQL/BIGR on basic cluster so will tweak code to handle that case.

```
:ShowMaster

KNOX_GATEWAY (KNOX) hostname is: [bi-hadoop-prod-4094.services.dal.bluemix.net]
HIVE_SERVER (HIVE) hostname is: [bi-hadoop-prod-4094.services.dal.bluemix.net]

BUILD SUCCESSFUL
```

**Comment by snowch** [comment]
Great stuff - thanks Pierre.

**Comment by pregazzoni** [comment]
New implementation checks if service is present of will just return service not installed, for example:

```
:classes
:ShowMaster

Service BIGSQL not installed
KNOX_GATEWAY (KNOX) hostname is: [bi-hadoop-prod-4094.services.dal.bluemix.net]
HIVE_SERVER (HIVE) hostname is: [bi-hadoop-prod-4094.services.dal.bluemix.net]
Service BIGR not installed

BUILD SUCCESSFUL

Total time: 5.427 secs
```

**Comment by pregazzoni** [comment]
fix via commit 02f7a21
